src/NITTA/LuaFrontend.hs

```diff
@@ -15,461 +11,398 @@ Copyright   : (c) Aleksandr Penskoi, 2019
 License     : BSD3
 Maintainer  : aleksandr.penskoi@gmail.com
 Stability   : experimental
+
+This module analyzes an abstract syntax tree of the Lua language source code, provided by Language.Lua module,
+and stores it into a NITTA's data flow graph.
+
+Supported Lua costructions are:
+ - simple math operators (addition, subtraction, multiplication and division);
+ - variable assignments;
+ - bitwise left and right shifts;
+ - recursive calls.
```

Comment:
If I don’t mistake, that list will be squashed into one paragraph. Please, check it.